rclpy/rclpy/__init__.py

```diff
@@ -233,6 +237,10 @@ def create_node(
     :param use_global_arguments: ``False`` if the node should ignore process-wide command line
         arguments.
     :param enable_rosout: ``False`` if the node should ignore rosout logging.
+    :param rosout_qos_profile: A QoSProfile or a history depth to apply to rosout publisher.
+        In the case that a history depth is provided, the QoS history is set to KEEP_LAST,
+        the QoS history depth is set to the value of the parameter,
+        and all other QoS settings are set to their default values.
```

Comment:
One concern: changing QoS for rosout publisher would generate incompatible QoS setting with concerned subscribers. this possibly break the ROS 2 user application, this will not happen in downstream users, but we would want to add more comment for this?